frontend/src/scenes/persons/RelatedFeatureFlags.tsx

```diff
@@ -0,0 +1,188 @@
+import { LemonInput, LemonSelect, LemonTable, LemonTag, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { LemonTableColumns } from 'lib/components/LemonTable'
+import { normalizeColumnTitle } from 'lib/components/Table/utils'
+import { capitalizeFirstLetter } from 'lib/utils'
+import stringWithWBR from 'lib/utils/stringWithWBR'
+import React from 'react'
+import { urls } from 'scenes/urls'
+import { FeatureFlagReleaseType } from '~/types'
+import { relatedFeatureFlagsLogic, RelatedFeatureFlag } from './relatedFeatureFlagsLogic'
+
+interface Props {
+    distinctId: string
+}
+
+export enum FeatureFlagMatchReason {
+    ConditionMatch = 'condition_match',
+    NoConditionMatch = 'no_condition_match',
+    OutOfRolloutBound = 'out_of_rollout_bound',
+    NoGroupType = 'no_group_type',
+    Disabled = 'disabled',
+}
+
+const featureFlagMatchMapping = {
+    [FeatureFlagMatchReason.ConditionMatch]: 'Matches',
```

Comment:
@neilkakkar @clarkus We're only returned that it "matches" and not which filters it's matching from the API, which is why I've left it as "matches"

Reply:
Not sure if there are limiting reasons as to why the API doesn't return which set of filters the person belongs to @neilkakkar

Reply:
The API does return this! It's the `condition_index` part of `evaluation`.

Reply:
If condition_index is set, it refers to zero-indexed group that it matched.

Reply:
hahahah ok then

Reply:
done

<img width="1072" alt="Screen Shot 2022-09-07 at 5 17 05 PM" src="https://user-images.githubusercontent.com/25164963/188982719-06788065-a91e-42ad-9319-e49befb237e0.png">